tools/cellxgene_census_builder/src/cellxgene_census_builder/build_soma/experiment_builder.py

```diff
@@ -505,6 +594,10 @@ def dispatch_X_chunk(
     local_var_joinids = adata.var.join(global_var_joinids).soma_joinid.to_numpy()
     assert (local_var_joinids >= 0).all(), f"Illegal join id, {dataset_id}"
 
+    _is_spatial_assay = np.isin(adata.obs.assay_ontology_term_id.to_numpy(), ALLOWED_SPATIAL_ASSAYS)
+    if is_spatial_assay := _is_spatial_assay.any():
```

Comment:
Let's add a comment specifying that if this happens, the code will definitely need to be modified to account for the schema change.